apps/android/app/src/main/res/values-ru/strings.xml

```diff
@@ -58,18 +58,39 @@
 
     <!-- background service notice - SimpleXAPI.kt -->
     <string name="icon_descr_instant_notifications">Мгновенные уведомления</string>
-    <string name="private_instant_notifications">Приватные мгновенные уведомления!</string>
-    <string name="private_instant_notifications_disabled">Приватные уведомления выключены!</string>
+    <string name="service_notifications">Мгновенные уведомления!</string>
+    <string name="service_notifications_disabled">Мгновенные уведомления выключены!</string>
     <string name="to_preserve_privacy_simplex_has_background_service_instead_of_push_notifications_it_uses_a_few_pc_battery">Чтобы защитить ваши личные данные, вместо уведомлений от сервера приложение запускает <b>фоновый сервис <xliff:g id="appName">SimpleX</xliff:g></b>, который потребляет несколько процентов батареи в день.</string>
     <string name="it_can_disabled_via_settings_notifications_still_shown"><b>Он может быть выключен через Настройки</b> – вы продолжите получать уведомления о сообщениях пока приложение запущено.</string>
-    <string name="turn_off_battery_optimization">Для использования приватных уведомлений, пожалуйста отключите оптимизацию батареи для <xliff:g id="appName">SimpleX</xliff:g> в слеующем диалоге. Иначе уведомления будут выключены.</string>
-    <string name="turning_off_background_service">Оптимизация батареи включена, поэтому приватные уведомления выключены. Вы можете снова включить их через Настройки.</string>
+    <string name="turn_off_battery_optimization">Для использования этой функции, пожалуйста, отключите оптимизацию батареи для <xliff:g id="appName">SimpleX</xliff:g> в следующем диалоге. Иначе уведомления будут выключены.</string>
+    <string name="turning_off_service_and_periodic">Оптимизация батареи включена, поэтому сервис уведомлений и периодические запросы новых сообщений выключены. Вы можете снова включить их через Настройки.</string>
+    <string name="periodic_notifications">Периодические уведомления</string>
+    <string name="periodic_notifications_disabled">Периодические уведомления выключены!</string>
+    <string name="periodic_notifications_desc">Чтобы защитить ваши личные данные, вместо уведомлений от сервера приложение может периодически запускаться и получать новые сообщения. Это потребляет несколько процентов батареи в день.</string>
 
     <!-- SimpleX Chat foreground Service -->
     <string name="simplex_service_notification_title"><xliff:g id="appNameFull">SimpleX Chat</xliff:g> сервис</string>
     <string name="simplex_service_notification_text">Приём сообщений…</string>
     <string name="hide_notification">Скрыть</string>
 
+    <!-- Notifications -->
+    <string name="settings_notifications_mode_title">Сервис уведомлений</string>
+    <string name="settings_notification_preview_mode_title">Показывать уведомления</string>
+    <string name="notifications_mode_off">Включен только на переднем плане</string>
```

Comment:
```suggestion
    <string name="notifications_mode_off">Включен только когда приложение открыто</string>
```
foreground is already quite technical, it feels like the translation is borderline  understandable.. maybe I'm just not used to the term

Reply:
there's explanation though in footer so it may be ok